src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -352,19 +353,76 @@ public class IndexRenameChange : IndexChange
         public string OldIndexName { get; set; }
     }
 
-    public class TrafficWatchChange : DatabaseChange
+    internal class TrafficWatchHttpChange : TrafficWatchChangeBase
```

Comment:
This is a breaking change, no?

Reply:
Yes, but this is not used anywhere in the Client API